composer.json

```diff
@@ -3,11 +3,16 @@
     "description": "GraphQL module for the Laravel and Lumen PHP frameworks.",
     "type": "library",
     "require": {
-        "youshido/graphql": "^1.5"
+        "youshido/graphql": "^1.5",
+        "jalle19/laravel-lost-interfaces": "^1.0",
+        "digiaonline/json-helpers": "^1.0",
+        "php": ">=7.1"
```

Comment:
Can you add `"sort-packages": true` to the `config` section?

Reply:
Once you've added sort-packages, can you require https://packagist.org/packages/illuminate/contracts as a dependency? We use for some excpetions and what not, and requiring a new package will also cause Composer to sort the list of packages once.

Reply:
Actually add `illuminate/console` and `illuminate/support` too.